### ISSUE TITLE
missing fx_media_open_notify initialization without FX_DISABLE_FORCE_MEMORY_OPERATION defined

### ISSUE THREAD

**Issue by TheFlyvio** [azure-rtos/filex#41]
when user does not need the open notification, the FX_DISABLE_FORCE_MEMORY_OPERATION is undefined and fx_media_open_notify_set(fx_media, NULL) is not called before the opening, the FX_MEDIA fx_media_open_notify attribute is never initialized, resulting in a potential hard fault.

Azure RTOS 6.3.0

1. let FX_DISABLE_FORCE_MEMORY_OPERATION  undefined
2. do not call fx_media_open_notify_set
3. create FX_MEDIA in the stack after some work was done and memory isnt only zeroes anymore
4. open the media
5. the "address" contained in fx_media_open_notify will be called

two ways it can be worked around 
-defining FX_DISABLE_FORCE_MEMORY_OPERATION  
-before the opening, calling fx_media_open_notify_set with a function pointer or null

**Comment by xiuwencai** [comment]
Yes, this is a bug. Another workaround is defining FX_MEDIA as a global variable so that it contains all zeros.
